src/algorithms/reco/JetReconstruction.cc

```diff
@@ -23,31 +23,32 @@ namespace eicrecon {
     m_log->trace("Initialized");
   }
 
-  std::vector<edm4eic::ReconstructedParticle*> JetReconstruction::execute(
+  edm4eic::ReconstructedParticleCollection* JetReconstruction::execute(
```

Comment:
```suggestion
  std::unique_ptr<edm4eic::ReconstructedParticleCollection> JetReconstruction::execute(
```

Reply:
Whoops! Thank you for catching that: I deleted the wrong line while cleaning up.